OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -10,7 +10,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Taʾammǝra Māryām, Taʾammǝra ʾIyasus, Homily by Jacob of Sarug on the Angel and the Thief</title>
+                <title xml:lang="gez">ተአምረ፡ ማርያም፡, ተአምረ፡ ኢየሱስ፡, ድርሳን፡ ቃሉ፡ ዘቅዱስ፡ ወብጹዕ፡ ያዕቆብ፡ ኤጲስ፡ ቆጶስ፡ ዘሥሩግ፡ በእንተ፡ መልአክ፡ ወፈያታዊ፡</title>
+                <title xml:lang="gez" type="normalized">Taʾammǝra Māryām, Taʾammǝra ʾIyasus, Dǝrsān qālu za-bǝṣuʿ Yāʿqob ʾeṗis qoṗos za-Śǝrug baʾǝnta malʾak wa-fayyātāwi</title>
+                <title xml:lang="en">Hymns, New Testament excerpts, Miracles of Mary, Miracles of Jesus, Homily by the blessed Yāʿqob, bishop of Śǝrug on the Angel and the Thief</title>
```

Comment:
Clearly, the suggested title format does not work well in cases like this, where works that are not all listed under a known Geez name are contained in the manuscripts. Do you have suggestions?

Reply:
this is a point we should perhaps discuss again in a meeting. Whatever solution we adopt, I fear that a certain degree of incoherence seems unavoidable (like in the solution here proposed). If we choose to omit some works (e.g. here hymns and biblical readings), we should also avoid to do it in some mss but not in others. For the moment we might keep this ms as a good example of this issue and compare it with additional examples in our catalogue.